src/views/Simulation/components/Scene/Scene.js

```diff
@@ -59,6 +60,30 @@ const Scene = ({ setSelectedElement }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const centerToPosition = useCallback(
+    () => (elementId) => {
+      if (sceneContainerRef.current == null) return;
+      const newSelectedElement = sceneContainerRef.current.findElementById(elementId);
+      sceneContainerRef.current.centerOnElement(newSelectedElement);
+    },
+    [sceneContainerRef]
+  );
+
+  useEffect(() => {
+    setCenterToPosition(centerToPosition);
+  }, [centerToPosition, setCenterToPosition]);
+
+  useEffect(() => {
+    if (sceneContainerRef.current == null) return;
+    const newSelectedElement = sceneContainerRef.current.findElementById(selectedElementId);
+    sceneContainerRef.current.highlightElement(newSelectedElement);
+    return () => {
+      const lastSelectedElement = sceneContainerRef.current.findElementById(selectedElementId);
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      sceneContainerRef.current.unhighLightElement(lastSelectedElement);
```

Comment:
There is still a capital 'L' that shouldn't be here :wink: 
(`unhighLightElement` -> `unhighlightElement`)